packages/react-ux-capture-example/src/Foo.jsx

```diff
@@ -14,25 +36,97 @@ const Foo = () => (
 		primaryActionAvailable={primaryActionAvailable}
 	>
 		<div className="chunk">
-			<h1 className="text--pageTitle">Foo Title</h1>
-			<Inline mark="ux-text-foo-title" />
+			<h1
+				className="text--pageTitle"
+				style={getBoxStyle('ux-destination-verified')}
+			>
+				Foo Title
+			</h1>
+			<UXCaptureInlineMark mark="ux-text-foo-title" />
 		</div>
 		<div className="chunk">
-			<p>
-				Primary content paragraph. All content in this view is loaded
-				synchronously - the measures correspond to the client-side app
-				rendering time
+			<p style={getBoxStyle('ux-primary-content-displayed')}>
+				Primary content paragraph. Click action for the button still takes
+				time to attach and content for the rest of the view is lazy-loaded.
 			</p>
-			<Inline mark="ux-text-foo-primary" />
+			<UXCaptureInlineMark mark="ux-text-foo-primary" />
 		</div>
 		<div className="chunk">
-			<button>Primary action button</button>
-			<Inline mark="ux-button-foo-primary" />
+			<Lazy
+				delay={1000}
+				fallback={
+					<React.Fragment>
+						<button
+							className="button"
+							disabled
+							style={getBoxStyle('ux-primary-action-available')}
+						>
+							Primary action button
+						</button>
+						<UXCaptureInlineMark mark="ux-text-button" />
```

Comment:
I think you could put `<UXCaptureInlineMark />` inside `<button>` in order to avoid the `<React.Fragment>` parent

Reply:
Not sure if <script> tag is allowed inside a button (e.g. on SSR).

Reply:
Let's get back to this when we SSR, right now, I'd leave it as is implying that UXCaptureInlineMark actually renders something to the output.